packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3181,6 +3181,8 @@ export class ContainerRuntime
 			try {
 				summarizeResult = await this.summarize({
 					fullTree: fullTree || forcedFullTree,
+					//* Can't remember why this is needed but the test fails wihtout it
+					fullGC: fullTree || forcedFullTree,
```

Comment:
GC already takes of running fullGC if it's needed. So, it's not needed here. Also, fullTree doesn't necessarily mean fullGC.

Reply:
If I need it for test setup I'll use the GC Option